lib/spack/docs/build_systems/intelpackage.rst

```diff
@@ -0,0 +1,994 @@
+.. _intelpackage:
+
+.. contents::
+
+
+--------------------------
+Intel packages in Spack
+--------------------------
+
+Spack can work with several software development products offered by Intel.
+Some of these are available under no-cost terms, others require a paid license.
+All share the same basic steps for configuration, installation, and, where
+applicable, license management. The relevant Spack Python class is:
+
+.. code-block:: python
+
+  IntelPackage
+
+Spack handles several interaction routes with Intel tools, like it does for any
```

Comment:
I introduced this term because the complexity of the Studio package dictates fairly different steps for those "interaction routes". I felt I needed a suitable and specific term to differentiate and refer to them.

Of course, to no small extent, this view reflects my own path to understanding how Spack works. To show them clearly should help, _especially_ for beginners who are hit with _all of this upfront_ when they "simply" wish to use Intel tools.